### PR TITLE
refactor: use "payout" rather than "price" in PrePOMarket.sol

### PR DESCRIPTION
**Pull request by davidprepo**
Might have been a bit overzealous with this, chased this down the rabbithole and converted to "payout" everywhere which seemed relevant.

### CODE REVIEW

File: apps/smart-contracts/core/contracts/interfaces/IPrePOMarket.sol [hunk 1]
```diff
@@ -151,39 +151,39 @@ interface IPrePOMarket {
   function getShortToken() external view returns (ILongShortToken);
 
   /**
-   * @notice Returns the lower bound of what a Long token can be priced at
+   * @notice Returns the lower bound of what a Long token can be payoutd at
```
Comment (XavierEkkel) [inline apps/smart-contracts/core/contracts/interfaces/IPrePOMarket.sol:154]: make sure to manually check too if doing find+replace all
```suggestion
   * @notice Returns the lower bound of what a Long token can be paid out at
```

File: apps/smart-contracts/core/contracts/interfaces/IPrePOMarket.sol [hunk 2]
```diff
@@ -151,39 +151,39 @@ interface IPrePOMarket {
...
-   * @return Minimum Long token price
+   * @dev Must be less than ceilingLongPayout and MAX_PAYOUT.
+   * @return Minimum Long token payout
    */
-  function getFloorLongPrice() external view returns (uint256);
+  function getFloorLongPayout() external view returns (uint256);
 
   /**
-   * @notice Returns the upper bound of what a Long token can be priced at
+   * @notice Returns the upper bound of what a Long token can be payoutd at
```
Comment (XavierEkkel) [inline apps/smart-contracts/core/contracts/interfaces/IPrePOMarket.sol:162]: same here

**Review by XavierEkkel — APPROVED** [review]
LGTM after nits

**Review by ramenforbreakfast — APPROVED** [review]
LGTM

**Pull request merged**